#请求播放语音消息

string  audioFilePath    #声音文件路径
string  codecFormat      #语音文件编码格式，如"mp3"、"wav"等 
int32   order		     #命令，0：播放语音文件、1：停止发言、2：允许发言

